### ISSUE TITLE
[Linter] Error running Unknown Error: Process exited with no output, code: 0

### ISSUE THREAD

**Issue by odevices** [AtomLinter/linter-lua#46]
Hello.

I've been running linter-lua with no problems for months now but when I updated today to 1.0.3, I get the following error:

```
linter-registry.js:144 
[Linter] Error running Unknown Error: Process exited with no output, code: 0
    at ChildProcess.<anonymous> (C:\Users\clarkson\.atom\packages\linter-lua\node_modules\sb-exec\lib\index.js:67:20)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:877:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:226:5)
```

Oddly enough, linter will run once successfully but then after that the above error occurs repeatedly everytime linter runs (e.g. when I make a change to a file).

I'm running on Windows 7 and I'm using lua 5.3 (luac.exe).

**Comment by Arcanemagus** [comment]
Ah, gotta love providers with no specs to catch things like this... I'll push out a fix to this shortly!

**Comment by Arcanemagus** [comment]
Fixed in v1.0.4, thanks for reporting this!

**Comment by odevices** [comment]
Thank you!
